textBasedAdventure.js

```diff
@@ -1,5 +1,169 @@
 const readline = require('readline-sync')
 
+//Everyone's health and attack declared
+let warriorHp = 35
+let warriorAtt = Math.floor(Math.random() * 20);
+let minion1Hp = 15
+let minion1Att = Math.floor(Math.random() * 7);
+let minion2Hp = 20
+let minion2Att = Math.floor(Math.random() * 10);
+let minotaurHp = 40
+let minotaurAtt = Math.floor(Math.random() * 15);
+let Inventory = 2
+let potionUse = 20
+//Delay in terminal
+function wait(ms){
+    var start = new Date().getTime();
+    var end = start;
+    while(end < start + ms) {
+      end = new Date().getTime();
+   }
+ }
 let nameInput = readline.question("Enter your name: ")
+wait(1500)
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+console.log(`Hello ${nameInput}!  Welcome to Jovanni's RPG retro game`)
+wait(1500)
+
+let classInput = readline.question('Choose a class: Warrior or Mage or Archer? '); wait(1500); {
+    if(classInput === "warrior" || classInput === "warrior") wait(1500); {
```

Comment:
No need to check with the `if` statement because the outcome is the same no matter what the user chose!